antarest/study/business/xpansion_management.py

```diff
@@ -381,49 +388,50 @@ def update_xpansion_constraints_settings(
         xpansion_settings = XpansionSettingsUpdate.model_validate(args)
         return self.update_xpansion_settings(study, xpansion_settings)
 
-    def _add_raw_files(
-        self,
-        file_study: FileStudy,
-        files: List[UploadFile],
-        resource_type: XpansionResourceFileType,
-    ) -> None:
-        keys = get_resource_dir(resource_type)
-        data: JSON = {}
-        buffer = data
-
-        list_names = [file.filename for file in files]
-        for name in list_names:
-            try:
-                if name in file_study.tree.get(keys):
-                    raise FileAlreadyExistsError(f"File '{name}' already exists")
-            except ChildNotFoundError:
-                logger.warning(f"Failed to list existing files for {keys}")
-
-        if len(list_names) != len(set(list_names)):
-            raise FileAlreadyExistsError(f"Some files have the same name: {list_names}")
-
-        for key in keys:
-            buffer[key] = {}
-            buffer = buffer[key]
-
-        for file in files:
-            content = file.file.read()
-            if isinstance(content, str):
-                content = content.encode(encoding="utf-8")
-            assert file.filename is not None
-            buffer[file.filename] = content
-
-        file_study.tree.save(data)
-
     def add_resource(
         self,
         study: StudyInterface,
         resource_type: XpansionResourceFileType,
-        files: List[UploadFile],
+        file: UploadFile,
     ) -> None:
```

Comment:
This method returns None whereas it is supposed to return "Any" in the API definition